setup.py

```diff
@@ -84,8 +84,11 @@ def find_any_distribution(pkgs):
     url='https://optuna.org/',
     packages=find_packages(),
     package_data={
-        'optuna':
-        ['storages/rdb/alembic.ini', 'storages/rdb/alembic/*', 'storages/rdb/alembic/versions/*']
+        'optuna': [
+            'storages/rdb/alembic.ini',
+            'storages/rdb/alembic/*.*',
+            'storages/rdb/alembic/versions/*.*'
+        ]
```

Comment:
This comment is just a clarification.
Currently, all target files of wildcard patterns have extensions like `.py` and `.mako` as follows:
- `optuna/storages/rdb/alembic/env.py`
- `optuna/storages/rdb/alembic/script.py.mako`
- `optuna/storages/rdb/alembic/versions/v0.9.0.a.py`

I'm wondering if there is a possibility to add files that have no extensions?

Reply:
> if there is a possibility to add files that have no extensions?

No.  The files that may be added in the future are only version files
(i.e., `optuna/storages/rdb/alembic/versions/*.py`).

Reply:
Thanks. I got it.